CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Guidelines for Contributing to Dafny
+
+Thanks for contributing to Dafny!  Github is the right place to discuss feature requests, report issues with Dafny, or contact the Dafny developers.
+Dafny is an Open Source project and welcomes contributions.
+
+## Reporting issues
+
+Before reporting an issue here, consider whether it would be better handled in one of the following places:
+
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/dafny), an online Q&A website, for help with writing and proving Dafny programs.
+- [`dafny-lang/ide-vscode`](https://github.com/dafny-lang/ide-vscode), Dafny's VSCode plugin, for issues or feature requests specific to the plugin itself and issues with Dafny's LSP server.
```

Comment:
Don't we still want to manage issues/requests for the LSP server here (as in the old version)?

Reply:
Ah - the original text was ambiguous (at least to me) and I clarified it in the wrong direction.

Reply:
Fixed.